docs/flowcollector-flows-netobserv-io-v1beta2.adoc

```diff
@@ -377,7 +377,7 @@ Type::
 
 | `action`
 | `string`
-| `action` defines the action to perform on the flows that match the filter. The available options are `Accept` and `Reject`.
+| `action` defines the action to perform on the flows that match the filter. The available options are `Accept` (default) and `Reject`.
```

Comment:
`action` defines the action to perform on the flows that match the filter. The available options are `Accept`, which is the default, and `Reject`.

Reply:
can we add the default info in parentheses? Sorry for the nit but IBMSG doesn't like parentheses 👼